code/modules/power/antimatter/shielding.dm

```diff
@@ -27,20 +27,28 @@
 	var/coredirs = 0
 	var/dirs = 0
 
+	var/destroy_message
 
 /obj/machinery/am_shielding/Initialize()
 	. = ..()
+	destroy_message = "<span class='danger'>The [name] melts!</span>"
 	addtimer(CALLBACK(src, .proc/controllerscan), 10)
 
 
 /obj/machinery/am_shielding/proc/controllerscan(priorscan = 0)
 	//Make sure we are the only one here
 	if(!istype(src.loc, /turf))
+		new/obj/item/device/am_shielding_container(src.loc)
+		visible_message("This should never happen! src.loc wasn't a type of /turf in shielding.dm")

```

Comment:
theres a different proc for this IIRC

Reply:
you mean the new/obj...? not sure about that, I just used that because that's all I know

Reply:
no. visible message. this is an error message so it shouldnt appear to the player